v2/emailpassword/common-customizations/email-verification/about.mdx

```diff
@@ -147,6 +151,25 @@ SuperTokens.init({
     Session.init(),
   ],
 });
+
```

Comment:
we need to ask the user if they are using react router dom or not, just like we do in quick setup for frontend